security/s2a/internal/crypter/expander_test.go

```diff
@@ -0,0 +1,66 @@
+package crypter
+
+import (
+	"bytes"
+	"crypto/sha256"
+	"google.golang.org/grpc/security/s2a/internal/crypter/testutil"
+	"testing"
+)
+
+type hkdfTestVector struct {
+	comment        string
```

Comment:
Let's call this `desc`.

Also, sorry for missing that in the PR #8. Can you please send a separate PR to change `comment` to `desc`?

Reply:
Done. Will change the rest in a separate PR.